source/channels/message-priority.rst

```diff
@@ -0,0 +1,46 @@
+Set message priority (beta)
+===========================
+
+.. include:: ../_static/badges/allplans-cloud-selfhosted.rst
+  :start-after: :nosearch:
+
+.. |priority-icon| image:: ../images/Priority-Message-Icon.svg
+  :alt: Select the Priority icon to mark new messages as important or urgent before sending them.
+
+.. |acknowledge-button| image:: ../images/Ack-Button-Default.svg
+  :alt: Select the Acknowledge button to indicate that you've read it and taken necessary action.
+
+.. |acknowledged-icon| image:: ../images/Ack-Button-Active.svg
+  :alt: Hover over the Acknowledged icon to see who has read and actioned the message. Select the icon to remove your acknowledgement within five minutes.
+
+From Mattermost v7.6, you can add a message priority label to root messages to make important messages requiring timely action or response more visible and less likely to be overlooked.
+
+.. image:: ../images/Message-Priority-Animated-optimized.gif
+  :width: 700
+  :alt: Ensure important and urgent messages stand out clearly by adding priority labels to root messages.
+
+Set message priority
```

Comment:
Mobile can do this too, so we  should probably take the new tabbed approach to this content. cc// @cwarnermm @Rishselva